omero/users/cli/index.txt

```diff
@@ -5,6 +5,11 @@ The |CLI| is a set of Python_ based system-administration, deployment and
 advanced user tools. Most of commands work remotely so that the |CLI| can be
```

Comment:
They aren't brackets, it's mark up which gives mouseover text to define CLI as 'Command line interface' in the webpages